provisioning/decoder/idecoder_manager.go

```diff
@@ -0,0 +1,9 @@
+package decoder
+
+type IDecoderManager interface {
+	Init(dir string) error
+	Term() error
```

Comment:
Perhaps for consistency we can also re-name Term () to Close() ??

Reply:
Done in 9e7e332